pose_calculator.py

```diff
@@ -135,21 +136,24 @@ def get_values(self, display=True):
         frame, _ = self.generator.generate()
         contours, mask = self.detector.run_detector()
 
+        if contours is None:
+            return [None, None, None], [None, None, None]
+
         if len(contours) < 1:
             self.display_windows(frame, mask)
-            return [360, 360, 360], -1
+            return [None, None, None], [None, None, None]
 
         # sort contours by area in descending order
         contours.sort(key=lambda c: cv2.contourArea(c), reverse=True)
         c = contours[0]  # use the largest contour
 
         if cv2.contourArea(c) < 100:
             self.display_windows(frame, mask)
-            return [360, 360, 360], -1
+            return [None, None, None], [None, None, None]
 
         corners = self.get_corners(c)
         area = cv2.contourArea(c)
-        cv2.drawContours(frame, [corners], -1, (0, 0, 255), 6)
+        cv2.drawContours(frame, [corners], -1, (0, 0, 255), 2)
```

Comment:
What does this change do?

Reply:
It changes the line width of the vision target outline